ddtrace/debugging/_signal/snapshot.py

```diff
@@ -47,13 +47,25 @@ def timeout(_):
 
         return {
             "arguments": {
-                n: utils.capture_value(v, limits.max_level, limits.max_len, limits.max_size, limits.max_fields, timeout)
+                n: (
+                    utils.capture_value(
```

Comment:
should have utils.capture_value_and_redact method instead of duplicate this logic for both arguments and locals?